.eslintrc.cjs

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2024: true,
```

Comment:
The ECMAScript version 'es2024' is not valid. You should use a valid ECMAScript version like 'es2021' or 'es2022'.